workers/downloaders/tasks.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import absolute_import, unicode_literals
+from celery import shared_task
+
+from bioinformatics_mill_models.models import Batch
+
+
+@shared_task
+def save_test_batch():
+    print("saving batch")
```

Comment:
Yes. I agree wholeheartedly. I haven't gotten to handling logging properly yet but it's on my TODO list.